### PR TITLE
MM-1898 Updates to Mac OSX instructions

### PR DESCRIPTION
**Pull request by it33**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,12 +31,12 @@ Local Machine Setup (Docker)
...
-1. Follow the instructions at: http://docs.docker.com/installation/mac/  
-    1. Use the Boot2Docker command-line utility.
-    2. If you do command-line setup use: `boot2docker init eval “$(boot2docker shellinit)”`  
+1. Install Boot2Docker using instructions at: http://docs.docker.com/installation/mac/  
+    1. Start Boot2Docker from the command line and run: `boot2docker init eval “$(boot2docker shellinit)”`  
 2. Get your Docker IP address with: `boot2docker ip`
-3. Add a line to your /etc/hosts that goes: `<Docker IP> dockerhost`
-4. Run: `boot2docker shellinit` and copy the export statements to your ~/.bash\_profile.
+3. Use `sudo nano /etc/hosts` to add `<Docker IP> dockerhost` to your /etc/hosts file 
+4. Run: `boot2docker shellinit` and copy the export statements to your ~/.bash\_profile by running `sudo nano ~/.bash_profile`. If you don't yet have a ~/.bash\_profile file this will create one for you.
```
Comment (nickago) [inline README.md]: Why the \ in '~/.bash_profile'?

**Comment by rgarmsen2295** [comment]
Other than @nickago's comment the changes look good.  However you might just leave that entire "If you don't have a bash profile..." sentence out just to keep the instructions as concise as possible, but not sure if that was a direct point of confusion during installation or not.

File: README.md (hunk 1 above)
Comment (crspeller) [inline README.md]: It is an escape character for markdown. In markdown \_  causes _italics_.

**Comment by crspeller** [comment]
I would have to agree with @rgarmsen2295. The line about creating .bash_profile is unnecessary.

**Comment by rgarmsen2295** [comment]
Looks good +1

**Comment by hmhealey** [comment]
Looks good

**Pull request merged**